tools/apiview/parsers/cpp-api-parser/ci.yml

```diff
@@ -118,12 +122,20 @@ stages:
                   - checkout: none
                   - download: current
                     artifact: release-artifacts
-                  - task: UniversalPackages@0
-                    displayName: Publish ApiView Parser.
+                  - pwsh: |
+                      $latestVersion = (azcopy list "https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-cpp/parseazuresdkcpp/") -replace '[\/;].*' | Get-Unique | sort {[version] $_} -Descending | Select-Object -First 1
+                      $major, $minor, $build = $latestVersion.Split('.')
+                      $nextVersion = [version]::new($major, 1 + $minor, $build).ToString()
+                      Write-Host "##vso[task.setvariable variable=APIVIEW_PARSER_VERSION]$nextVersion"
+                    displayName: Determine Next ApiView Parser Version
+                  - task: AzurePowerShell@5
```

Comment:
You can use this template https://dev.azure.com/azure-sdk/internal/_git/azure-sdk-build-tools?version=GBmaster&path=/pipelines/steps/binary-signing.yml to do the signing now.